.github/workflows/pypi-upload.yaml

```diff
@@ -23,5 +23,5 @@ jobs:
     - name: Publish to PYPI
       uses: pypa/gh-action-pypi-publish@master
       with:
-        user: ${{ secrets.PYPI_USERNAME }}
+        user: __token__
         password: ${{ secrets.PYPI_PASSWORD }}
```

Comment:
The token replaces the username but the password stays the same? Just want to make sure this isn't a typo, because I'm not sure I've ever seen a token used that way before.